src/components/axes/timeAxis.ts

```diff
@@ -240,8 +244,12 @@ export module Axis {
 
     public _computeHeight() {
       var textHeight = this._measurer.measure().height;
-      this._tierHeights = this._tierLabelPositions.map((pos: string) =>
-        textHeight + this.tickLabelPadding() + ((pos === "between") ? 0 : this._maxLabelTickLength()));
+      this._tierHeights = this._tierLabelPositions.map((pos: string, index: number) => {
+        if (index >= this._maximumTiers) {
+          return 0;
+        }
```

Comment:
Say someone sets the tier label positions, then sets the number of tiers in the configuration to a smaller number. Now they're out of sync...